projects/ngx-barcode6/package.json

```diff
@@ -21,8 +22,8 @@
     "url": "https://github.com/efgiese/ngx-barcode6/issues"
   },
   "peerDependencies": {
-    "@angular/common": ">=14.0.0 <17.0.0",
-    "@angular/core": ">=14.0.0 <17.0.0",
+    "@angular/common": ">=14.0.0 <17.0.4",
```

Comment:
It should be:
```suggestion
    "@angular/common": ">=14.0.0 <18.0.0",
```
Also we may be able to just have a minimum requirement:
```suggestion
    "@angular/common": ">=14.0.0",
```

Reply:
@dixy version constraint removed, thanks!